docs/source/user_guide/get_started.rst

```diff
@@ -8,55 +8,57 @@ a short example on how to use the plugin.
 Installation
 ++++++++++++
 
-The latest stable release of ``aiida-mlip``, including its dependencies, can be installed from PyPI by running:
+Create a new `virtual environment <https://docs.python.org/3/library/venv.html#creating-virtual-environments>`_ and activate it to run the commands below.
 
-.. code-block:: bash
+The latest stable release of ``aiida-mlip``, including its dependencies, can be installed from PyPI by running in the venv:
 
-    python3 -m pip install aiida-mlip
+.. code-block:: bash
 
+    pip install aiida-mlip
 
 To get all the latest changes, ``aiida-mlip`` can also be installed from GitHub:
 
 .. code-block:: bash
 
-    python3 -m pip install git+https://github.com/stfc/aiida-mlip.git
-
+    pip install git+https://github.com/stfc/aiida-mlip.git
 
 By default, no machine learnt interatomic potentials (MLIPs) will be installed with ``aiida-mlip``.
 However, ``aiida-mlip`` currently provides an ``extra``, allowing MACE to be installed:
 
 .. code-block:: bash
 
-    python3 -m pip install aiida-mlip[mace]
-
+    pip install aiida-mlip[mace]
 
 For additional MLIPs, it is recommended that the ``extra`` dependencies provided by ``janus-core`` are used.
 For example, to install CHGNet and SevenNet, run:
 
 .. code-block:: bash
 
-    python3 -m pip install janus-core[chgnet,sevennet]
-
+    pip install janus-core[chgnet,sevennet]
 
 Please refer to the ``janus-core`` `documentation <https://stfc.github.io/janus-core/getting_started/getting_started.html#installation>`_ for further details.
 
 Once ``aiida-mlip`` and the desired MLIP calculators are installed, run::
 
-    verdi quicksetup  # better to set up a new profile
+    verdi presto  # better to set up a new profile
     verdi plugin list aiida.calculations  # should now show your calculation plugins
 
 Then use ``verdi code setup`` with the ``janus`` input plugin
-to set up an AiiDA code for aiida-mlip.
+to set up an AiiDA code for aiida-mlip. This `notebook <https://github.com/stfc/aiida-mlip/blob/main/examples/tutorials/setup-janus-code.ipynb>`_ may be useful.
```

Comment:
Can we link https://aiida.readthedocs.io/projects/aiida-core/en/stable/howto/run_codes.html#how-to-create-a-code here?